package.json

```diff
@@ -14,15 +14,16 @@
   ],
   "scripts": {
     "dev": "vite",
-    "build": "tsc && npm run lint && vite build && typedoc src/index.ts",
+    "build": "tsc && npm run lint && vite build && npm run typedef",
     "preview": "vite preview",
     "prepare": "vite build",
     "typecheck": "tsc",
     "lint": "eslint 'src/**/*.{ts,tsx}' --cache",
     "clean": "rimraf dist",
     "prettier": "prettier '{src,spec}/**' --list-different",
     "prettier:fix": "prettier '{src,spec}/**' --write",
-    "test": "jest spec"
+    "test": "jest spec",
+    "typedef": "typedoc src/index.ts"
```

Comment:
typedef -> typedoc ?
otherwise this could be confused with typecheck (which can emit type definitions)